tooling/storybook-addon/withClassInject.tsx

```diff
@@ -0,0 +1,30 @@
+import { makeDecorator } from "@storybook/preview-api";
+import { useEffect, type ReactNode } from "react";
+
+const PROVIDER_CLASSNAME = "hop-HopperProvider";
+const DEFAULT_CLASSNAME = "hop";
+const STORYBOOK_CLASSNAME = "sb-show-main";
+
+export const withClassInject = makeDecorator({
```

Comment:
is this still necessary? even with the theme provider?

Reply:
no it's not necessary anymore, I'll delete the file